src/native/third_party/common_third_party.gypi

```diff
@@ -2,7 +2,7 @@
 {
     'includes': ['../common.gypi'],
     'variables': {
-        'disabled_warnings': ['-W', '-Wall'],
+        'disabled_warnings': ['-W', '-Wall', '-Werror'],
```

Comment:
Can you try to disable just `-Wextra` and `-Wpedantic` instead of disabling all warning-as-errors and see if that works for third_party? I would still rather the default `-Werror` to be applied for warnings that we choose to enable.